neovim-sys/src/api/vim/nvim_error.rs

```diff
@@ -0,0 +1,95 @@
+//!
+//! This module contains the wrapper and related functions for neovim's Lua `Error`.
+//!
+use std::{ffi::CStr, fmt, os::raw::c_char};
+
+/// Wrapper for neovim's Lua `Error`.
+///
+#[derive(thiserror::Error, Clone, Copy)]
+#[repr(C)]
+pub struct NvimError {
```

Comment:
Follow pattern from `String`/`LuaString`: rename to `Error` and export as `LuaError`.